src/plugins/email/email.c

```diff
@@ -0,0 +1,93 @@
+/**
+ * @file
+ *
+ * @brief Source for email plugin
+ *
+ * @copyright BSD License (see doc/LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#include <kdberrors.h>
+#include <regex.h>
+#include <stdio.h>
+
+#include "email.h"
+
+static int validateEmail (const char * addr)
+{
+	if (!addr) return 0;
+	const char * regexString =
```

Comment:
It would be nice, if the plugin would to some more complex validation than just a regex. A simple regex can already be used via the `validation` plugin. Admittedly, this regex is quite complex so adding it to every key that should be an email would be annoying.

Also the regex is probably fine for our purpose, but technically doesn't cover all valid email addresses (which isn't possible with a regx). See also [this video](https://www.youtube.com/watch?v=xxX81WmXjPg).
You should probably add a "Limitations" line to the README stating that the plugin covers "most but not all valid" email addresses.